src/Raven.Studio/typescript/common/shell/menu/items/settings.ts

```diff
@@ -163,7 +163,7 @@ function getSettingsMenuItem(appUrls: computedAppUrls) {
             title: 'Backup Task',
             nav: false,
             dynamicHash: appUrls.editExternalReplicationTaskUrl,
-            itemRouteToHighlight: 'databases/tasks/ongoingTasks'
+            itemRouteToHighlight: 'databases/tasks/backups'
```

Comment:
title : Backup Task dynamic hash: ExternalReplication, itemRouteToHighlight: backup. This is suspicious.

Reply:
fixed